keyboards/keebio/quefrency/keymaps/kingwangwong/rules.mk

```diff
@@ -0,0 +1 @@
+DISABLE_JTAG = yes
```

Comment:
This file can be deleted, `DISABLE_JTAG` does nothing (it should already be disabled by default).